src/assets/style-components/_about.scss

```diff
@@ -0,0 +1,98 @@
+@use './mixins' as *;
+@use './variables' as *;
+
+.about-container {
+    padding: 3rem;
+    background-color: var(--dark-blue);
+
+    @include large {
+        padding: 5rem;
+    }
+
+    .container {
+        margin-inline: auto;
+
+        display: flex;
```

Comment:
As a suggession; if you use mixins(in this case `flex())` we can prevent code duplication

Reply:
we both agreed to make a more flexible flex mixin in another branch that has default values.